### DISCUSSION TITLE
"Could not create IO object" error for T2

### DISCUSSION THREAD

**Discussion by ramBrain** [simnibs/simnibs#150] [7.Creating head models] [answered]
Dear all,

I am trying to run `charm` from Windows using T1 and T2, but it prompts this error:

```
RuntimeError: C:\Program Files (x86)\ITK\include\ITK-4.13\itkImageFileReader.hxx:143:
 Could not create IO object for reading file D:\<path>\sub-026\m2m_sub-026\T2_reg.nii.gz
The file doesn't exist.
```

It seems that `charm` does not copy and coregister the T2 in the result folder as it is supposed to do, but then looks for that file.

Any clue?

Thank you,
Ramtin

**Comment by oulap** [comment, marked as answer]
Yeah there was a bug in the release. I just fixed it yesterday and made a new pre-release. It will be a pre-release until we get the windows installer signed, but you can try to see if it installs without the signing. Windows will probably throw lots of warnings though. 

You can also install from source or using the wheel if you don't want to wait for the signing.

**Reply by ramBrain** [reply]
I see, thank you for the clarification!

Ramtin.
